### PR TITLE
New post: world-management

### PR DESCRIPTION
**Pull request by theTWIXhunter**

### CODE REVIEW

**Comment by theTWIXhunter** [comment]
wrong button sorry

**Comment by LiljaFi** [comment]
My opinion does not matter, but seems quite weird to ignore the Worlds page - perhaps make it as an primary option and file manager as alternative?

**Comment by theTWIXhunter** [comment]
good idea

**Comment by theTWIXhunter** [comment]
lemme go add that

**Comment by LiljaFi** [comment]
👍

**Comment by theTWIXhunter** [comment]
what about the SFTP part? (see [my message in the kb-thread](https://discord.com/channels/710503370187735160/1261685069353979975/1274764130292727818)

**Comment by Mocab** [comment]
As per the warning in both the dashboard and sftp guide, users should avoid sftp if possible. The sftp guide alone is sufficient if users still choose to use it.

**Comment by theTWIXhunter** [comment]
> As per the warning in both the dashboard and sftp guide, users should avoid sftp if possible. The sftp guide alone is sufficient if users still choose to use it.

Ok

**Comment by iamdeka** [comment]
> My opinion does not matter

😃

**Comment by theTWIXhunter** [comment]
ignore the last 2 commits (they are the wrong branch)

**Comment by theTWIXhunter** [comment]
I am still not happy about the paper part so let me know if anyone knows a better way to do this

**Comment by theTWIXhunter** [comment]
I am not sure how the world tab works... do you just select the world folder? what about paper worlds?
(can anyone explain? @Mocab

**Comment by Mocab** [comment]
> I am not sure how the world tab works... do you just select the world folder? what about paper worlds? (can anyone explain? @Mocab)

Afaik you can import worlds with the "import" button and for paper servers you can upload all three worlds set the overworld as the "primary" world. There also a place to edit world settings, gamerules and a place to download public worlds

**Comment by theTWIXhunter** [comment]
oh, the github remote repositories plugin (from VScode) finaly worked

**Comment by theTWIXhunter** [comment]
ignore the second commit (it shoud be the exact same)

**Comment by theTWIXhunter** [comment]
I think I am ready

File: content/_posts/falix/dashboard/2024-08-18-world-management.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+title: Learn how to upload and download your world
+category: Dashboard
+tags: Server
+permalink: /minecraft/java/general/world-management
+description: Learn how to upload and download your world.
+author: TheTWIXhunter
+toc: true
+---
+
+# how to upload a world to the server.
```
Comment (iamdeka) [inline content/_posts/falix/dashboard/2024-08-18-world-management.md:12]: capitalize

File: content/_posts/falix/dashboard/2024-08-18-world-management.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## How to upload your world using the worlds page
+
+1. Log in to the [Dashboard](https://client.falixnodes.net/).
+
+2. Scroll down and click on the server you want the world uploaded to.
+
+3. Navigate to the [worlds page](https://client.falixnodes.net/server/worlds) by hovering over "Server" and pressing "worlds".
```
Comment (iamdeka) [inline content/_posts/falix/dashboard/2024-08-18-world-management.md:20]: capitalize Worlds

File: content/_posts/falix/dashboard/2024-08-18-world-management.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+tags: Server
+permalink: /minecraft/java/general/world-management
+description: Learn how to upload and download your world.
+author: TheTWIXhunter
+toc: true
+---
+
+# how to upload a world to the server.
+
+## How to upload your world using the worlds page
```
Comment (iamdeka) [inline content/_posts/falix/dashboard/2024-08-18-world-management.md:14]: dont need to repeat yourself

File: content/_posts/falix/dashboard/2024-08-18-world-management.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+1. Log in to the [Dashboard](https://client.falixnodes.net/).
+
+2. Scroll down and click on the server you want the world uploaded to.
+
+3. Navigate to the [worlds page](https://client.falixnodes.net/server/worlds) by hovering over "Server" and pressing "worlds".
+
+4. Press the "upload world" button and upload the chosen world folder.
```
Comment (iamdeka) [inline content/_posts/falix/dashboard/2024-08-18-world-management.md:22]: capitalize

**Comment by theTWIXhunter** [comment]
Resolved all reviews (I think) (tell me when I missed something)

**Comment by theTWIXhunter** [comment]
please ignore the old /dashboard file

File: content/_posts/minecraft/java/2024-08-18-world-management.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Learn how to upload and download your world
+category: Dashboard
+tags: Server
+permalink: /minecraft/java/general/world-management
+description: Learn how to upload and download your world.
```
Comment (iamdeka) [inline content/_posts/minecraft/java/2024-08-18-world-management.md:7]: title and description are the same - this is bad

File: content/_posts/minecraft/java/2024-08-18-world-management.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+title: Learn how to upload and download your world
+category: Dashboard
+tags: Server
+permalink: /minecraft/java/general/world-management
+description: Learn how to upload and download your world.
+author: TheTWIXhunter
+toc: true
+---
+
+# How to upload a world to the server.
```
Comment (iamdeka) [inline content/_posts/minecraft/java/2024-08-18-world-management.md:12]: > the server
i think using "your server" will be better

**Review by Mocab — APPROVED** [review]

**Comment by Mocab** [comment]
Ready to merge?

**Pull request merged**